rotkehlchen/globaldb/handler.py

```diff
@@ -1048,3 +1048,51 @@ def get_historical_price_data(source: HistoricalPriceOracle) -> List[Dict[str, A
              'from_timestamp': entry[2],
              'to_timestamp': entry[3],
              } for entry in query]
+
+    @staticmethod
+    def rebuild_assets_list() -> Tuple[bool, str]:
+        """
+        Delete current information about assets and repopulate with the last
+        builtin version
+        """
+        root_dir = Path(__file__).resolve().parent.parent
+        builtin_database = root_dir / 'data' / 'global.db'
+        remove_assets = 'DELETE FROM assets;'
+        remove_tokens = 'DELETE FROM ethereum_tokens;'
```

Comment:
you are completely missing the `underlying_tokens_list` and the `common_asset_details` tables here.